web/integration-core/src/jsMain/kotlin/androidx/compose/web/sample/Sample.kt

```diff
@@ -258,23 +259,22 @@ fun MyInputComponent(text: State<String>, onChange: (String) -> Unit) {
                 }
                 onTextInput {
                     onChange(it.inputValue)
-                    println("On input = : ${it.nativeEvent.isComposing} - ${it.inputValue}")
                 }
                 onKeyUp {
                     println("On keyUp key = : ${it.getNormalizedKey()}")
                 }
             }
         )
     }
-    Div(
-        attrs = {
+    Div {
+        Input(type = InputType.Checkbox, attrs = {
```

Comment:
I suggest us to have standard definition for most popular inputs - I can clearly see one it will be more intuitive to write something like

```RadioInput({...})```

Reply:
Added them in `InputElements.kt` file.
Maybe we also want to add more attributes to the functions signatures to let set the attributes more easily.